docs/DXIL.rst

```diff
@@ -3066,7 +3066,7 @@ INSTR.TEXTUREOFFSET                       offset texture instructions must take
 INSTR.TGSMRACECOND                        Race condition writing to shared memory detected, consider making this write conditional.
 INSTR.UNDEFRESULTFORGETDIMENSION          GetDimensions used undef dimension %0 on %1.
 INSTR.WRITEMASKFORTYPEDUAVSTORE           store on typed uav must write to all four components of the UAV.
-INSTR.WRITEMASKMATCHVALUEFORUAVSTORE      uav store write mask must match store value mask, write mask is %0 and store value mask is %1.
+INSTR.WRITEMASKMATCHVALUEFORUAVSTORE      Assignment of undefined values to UAV.
```

Comment:
Couldn't this also occur if non-undef values are provided for components where the corresponding bit in the write mask is zero?  Then this message would be misleading, wouldn't it?

Reply:
By the way, I  don't think this is a scenario you can repro with pure HLSL, which is often the nature of validator testing.  You have to do some IR replacement like other tests do in ValidationTest.cpp.

Reply:
After reading this again, I think you are saying that the scenario described in your first comment is not possible using HLSL correct?

I suppose I could add an error for when the user-provided values are undefined using my new wording that only triggers when a bit is set in the write mask that isn't in the store value mask and keep the old error that triggers when that error doesn't and the masks differ.